src/omnisharp/server.ts

```diff
@@ -321,7 +321,7 @@ export class OmniSharpServer {
 
         try {
             let launchResult = await launchOmniSharp(cwd, args, launchInfo, this.platformInfo, options);
-            this.eventStream.post(new ObservableEvents.OmnisharpLaunch(launchResult.monoVersion, launchResult.command, launchResult.process.pid));
+            this.eventStream.post(new ObservableEvents.OmnisharpLaunch(launchResult.monoVersion, launchResult.command, launchResult.process.pid, options.monoPath));
```

Comment:
I think it would be better if the LaunchResult returned the monoLaunchPath(if any) that it used.

Reply:
Oh yes, that makes way more sense, not sure how I missed that. ☕️ ☕️ ☕️